models.yaml

```diff
@@ -0,0 +1,219 @@
+---
+models:
+- api_name: gpt-best
+  name: Best OpenAI Model
+  supports_images: true
+  supports_tools: true
+  input_token_cost_cents: '0.00025'
+  output_token_cost_cents: '0.001'
```

Comment:
Just like you left the quotes off the strings, I'm pretty sure you don't need quotes around floats in a yml file

Reply:
TIL a while back - if you convert `BigDecimal` (`.to_d`) -> YAML, it stores it as a string so it can record large floats; and if its a normal `float` (`.to_f`) -> YAML, then you get a normal float value.

So these strings are perhaps because the original "floating point number" was a BigDecimal

From schema.rb:

```ruby
    t.decimal "input_token_cost_cents", precision: 30, scale: 15
    t.decimal "output_token_cost_cents", precision: 30, scale: 15
```

Reply:
(note, I didn't write this YAML file, it was generated from the rake tasks)

Reply:
oh interesting, that’s a good lesson for me